site/docs/pages/token/token-selector.mdx

```diff
@@ -1,107 +1,129 @@
-    style="width: 16px; height: 16px; border-radius: 50%; overflow: hidden; background: blue;"
-  /><span
-    data-testid="ockTokenSelector_Symbol"
-    style="color: rgb(10, 11, 13); font-size: 16px; line-height: 1.5; font-weight: 500; margin-left: 4px;"
-    >ETH</span
+<button data-testid="ockTokenSelector_Button" class="ock-tokenselector-button">
+  <span class="ock-tokenselector-label">Select</span>
+  <svg
+    data-testid="ockTokenSelector_CaretUp"
+    width="16"
+    height="16"
+    viewBox="0 0 16 16"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
   >
-  <span style="margin-left: 8px;">
-    <svg width="16" height="17" viewBox="0 0 16 17" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <path
-        d="M12.95 4.85999L8.00001 9.80999L3.05001 4.85999L1.64001 6.27999L8.00001 12.64L14.36 6.27999L12.95 4.85999Z"
-        fill="#0A0B0D"
-      ></path>
-    </svg>
-  </span>
+    <path
+      d="M3.05329 10.9866L7.99996 6.03997L12.9466 10.9866L14.1266 9.80663L7.99996 3.67997L1.87329 9.80663L3.05329 10.9866Z"
+      fill="#0A0B0D"
+    ></path>
+  </svg>
 </button>
+<div class="ock-tokenselectordropdown-container">
+  <div class="ock-tokenselectordropdown-scroll">
+    <button data-testid="ockTokenRow_Container" class="ock-tokenrow-button">
+      <span class="ock-tokenrow-left">
+        <span class="ock-tokenrow-body">
+          <span class="ock-tokenrow-name">Ethereum</span>
+          <span class="ock-tokenrow-symbol">ETH</span>
+        </span>
+      </span>
+      <span data-testid="ockTokenRow_Amount" class="ock-tokenrow-data"></span>
+    </button>
+    <button data-testid="ockTokenRow_Container" class="ock-tokenrow-button">
+      <span class="ock-tokenrow-left">
+        <span class="ock-tokenrow-body">
+          <span class="ock-tokenrow-name">USDC</span>
+          <span class="ock-tokenrow-symbol">USDC</span>
+        </span>
+      </span>
+      <span data-testid="ockTokenRow_Amount" class="ock-tokenrow-data"></span>
+    </button>
+    <button data-testid="ockTokenRow_Container" class="ock-tokenrow-button">
+      <span class="ock-tokenrow-left">
+        <span class="ock-tokenrow-body">
+          <span class="ock-tokenrow-name">Dai</span>
+          <span class="ock-tokenrow-symbol">DAI</span>
+        </span>
+      </span>
+      <span data-testid="ockTokenRow_Amount" class="ock-tokenrow-data"></span>
+    </button>
+  </div>
+</div>
 ```
 
 :::
 
-<App>
-  <TokenSelector
-    token={{
-      address: '0x1234',
-      chainId: 1,
-      decimals: 18,
-      image:
-        'https://dynamic-assets.coinbase.com/dbb4b4983bde81309ddab83eb598358eb44375b930b94687ebe38bc22e52c3b2125258ffb8477a5ef22e33d6bd72e32a506c391caa13af64c00e46613c3e5806/asset_icons/4113b082d21cc5fab17fc8f2d19fb996165bcce635e6900f7fc2d57c4ef33ae9.png',
-      name: 'Ethereum',
-      symbol: 'ETH',
-    }}
-    onClick={() => {}}
-  />
-</App>
-
-`TokenSelector` with no token prop
-
-:::code-group
+## Props
```

Comment:
Had to remove the code here. This needs to be put back after a new release has been cut

```
<App>
  <TokenSelectorContainer>
    {(token, setToken) => (
      <TokenSelector token={token}>
        <TokenSelectorDropdown
          setToken={setToken}
          options={[
            {
              name: 'Ethereum',
              address: '',
              symbol: 'ETH',
              decimals: 18,
              image: 'https://wallet-api-production.s3.amazonaws.com/uploads/tokens/eth_288.png',
              blockchain: 'eth',
              chainId: 8453,
            },
            {
              name: 'USDC',
              address: '0x833589fcd6edb6e08f4c7c32d4f71b54bda02913',
              symbol: 'USDC',
              decimals: 6,
              image:
                'https://d3r81g40ycuhqg.cloudfront.net/wallet/wais/44/2b/442b80bd16af0c0d9b22e03a16753823fe826e5bfd457292b55fa0ba8c1ba213-ZWUzYjJmZGUtMDYxNy00NDcyLTg0NjQtMWI4OGEwYjBiODE2',
              blockchain: 'eth',
              chainId: 8453,
            },
            {
              name: 'Dai',
              address: '0x50c5725949a6f0c72e6c4a641f24049a917db0cb',
              symbol: 'DAI',
              decimals: 18,
              image:
                'https://d3r81g40ycuhqg.cloudfront.net/wallet/wais/d0/d7/d0d7784975771dbbac9a22c8c0c12928cc6f658cbcf2bbbf7c909f0fa2426dec-NmU4ZWViMDItOTQyYy00Yjk5LTkzODUtNGJlZmJiMTUxOTgy',
              blockchain: 'eth',
              chainId: 8453,
            },
          ]}
        />
      </TokenSelector>
    )}
  </TokenSelectorContainer>
</App>
```